web_src/js/index.js

```diff
@@ -2,6 +2,7 @@
 /* exported timeAddManual, toggleStopwatch, cancelStopwatch, initHeatmap */
 /* exported toggleDeadlineForm, setDeadline, updateDeadline, deleteDependencyModal, cancelCodeComment, onOAuthLoginClick */
 
+import 'jquery.are-you-sure';
```

Comment:
Why doesn't this require `./` like the ones below?

Reply:
In JS, `./` is a relative import. If it's missing, it refers to a module in `node_modules`.

Reply:
Actually this `./` mechanism is node.js/webpack specific. The web version of `import` functions differently but as long as we don't compile to JS module syntax, it will not be used.